test/reducers/players.js

```diff
@@ -68,6 +68,14 @@ describe('players reducer', () => {
     expect(playerState.playerId1.deck).to.have.lengthOf(24);
     expect(playerState.playerId1.hand).to.eql(['c25', 'c26', 'c27', 'c28', 'c29', 'c30']);
   });
+  it('should handle GAIN_MANA', () => {
+    const action1 = gainMana('playerId1', 1);
+    const action2 = gainMana('playerId2', 11);
+    let playerState = players(initialPlayerState, action1);
+    expect(playerState.playerId1.mana).to.eql(1);
```

Comment:
This isn't a big deal but you should use `.to.equal()` for testing primitives and `.to.eql()` for objects.